### PR TITLE
ANC Labs - Case management

### PR DESCRIPTION
**Pull request by anvmn**

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2021-12-08 14-09

**Comment by anvmn** [comment]
Selection_388

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Backend/Update.elm [hunk 1]
```diff
@@ -3732,6 +3848,66 @@ generatePrenatalAssesmentMsgs currentDate language isChw updateAssesment after i
...
+                                measurement.value
+                                    |> (\value ->
+                                            { value
+                                                | performedTests =
+                                                    if testExecuted then
+                                                        EverySet.insert test value.performedTests
+
+                                                    else
+                                                        EverySet.remove test value.performedTests
+                                                , resolutionDate = Date.add Days (prenatalLabExpirationPeriod + 1) currentDate
```
Comment (mariano-dagostino) [inline client/src/elm/Backend/Update.elm:3886]: :+1:

File: client/src/elm/Pages/AcuteIllnessActivity/View.elm [hunk 2]
```diff
@@ -3169,7 +3169,7 @@ viewContactsTracingFormRecordContactDetails language currentDate personId db dat
                             ToggleContactsTracingDateSelector
                             SetContactsTracingDate
                             data.isDateSelectorOpen
-                            (Date.add Days -10 currentDate)
+                            (Date.add Days (-1 * covidIsolationPeriod) currentDate)
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/AcuteIllnessActivity/View.elm:3172]: :+1:

**Review by mariano-dagostino — APPROVED** [review]

**Pull request merged**